src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryMessages.java

```diff
@@ -17,6 +17,5 @@
   public static final String SOME_VALID_WIDGETS_WERE_NOT_SWAPPED_MSG =
       "some valid widget placeholders are not swapped for widget inline frames";
 
-  private MercuryMessages() {
-  }
+  private MercuryMessages() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)